package.json

```diff
@@ -26,5 +27,8 @@
     "typescript": "^5.5.3",
     "typescript-eslint": "^8.0.1",
     "vite": "^5.4.1"
+  },
+  "resolutions": {
+    "styled-components": "^6"
```

Comment:
This ensures that we never accidentally install a version of styled components below major version 6.  The styled components docs recommended this (first time I've used this)